[BUG] App crashes on sending contacts as attachments

Rocket.Chat.Android version: (make sure you are running the latest)
<!-- Version can be found by opening the side menu and then clicking on the chevron alongside username -->
2.0.0-alpha1
<!-- Found a bug? List all devices that reproduced it and all that doesn't -->
Mobile device model and OS version: (e.g. "Nexus 7 - Android 6.0.1")
<!-- Don't forget to list the steps to reproduce. Stack traces may help too :) -->
Nexus 6 - Android 7.1

## Description
App crashes while selecting contacts in file attachments.
If contacts support is not there then the option should not be there to select.

### Logs
```
           CursorWindow  E  Failed to read row 0, column -1 from a CursorWindow which has 1 rows, 36 columns.
         AndroidRuntime  E  FATAL EXCEPTION: main
                         E  Process: chat.rocket.android.dev, PID: 26031
                         E  java.lang.IllegalStateException: Couldn't read row 0, col -1 from CursorWindow.  Make sure the Cursor is initialized correctly before accessing data from it.
                         E      at android.database.CursorWindow.nativeGetString(Native Method)
                         E      at android.database.CursorWindow.getString(CursorWindow.java:438)
                         E      at android.database.AbstractWindowedCursor.getString(AbstractWindowedCursor.java:51)
                         E      at android.database.CursorWrapper.getString(CursorWrapper.java:137)
                         E      at chat.rocket.android.util.extensions.UriKt.getFileName(Uri.kt:23)
                         E      at chat.rocket.android.chatroom.domain.UriInteractor.getFileName(UriInteractor.kt:15)
                         E      at chat.rocket.android.chatroom.presentation.ChatRoomPresenter$uploadFile$1$fileName$1.doResume(ChatRoomPresenter.kt:128)
                         E      at kotlin.coroutines.experimental.jvm.internal.CoroutineImpl.resume(CoroutineImpl.kt:54)
                         E      at kotlinx.coroutines.experimental.DispatchedTask$DefaultImpls.run(Dispatched.kt:161)
                         E      at kotlinx.coroutines.experimental.DispatchedContinuation.run(Dispatched.kt:25)
                         E      at java.util.concurrent.ForkJoinTask$RunnableExecuteAction.exec(ForkJoinTask.java:1383)
                         E      at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:256)
                         E      at java.util.concurrent.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1123)
                         E      at java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1961)
                         E      at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1909)
                         E      at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:128)

```

--- a/app/src/main/java/chat/rocket/android/chatroom/ui/ChatRoomFragment.kt
+++ b/app/src/main/java/chat/rocket/android/chatroom/ui/ChatRoomFragment.kt
@@ -467,6 +467,7 @@ class ChatRoomFragment : Fragment(), ChatRoomView, EmojiKeyboardListener, EmojiR
             val intent = Intent(Intent.ACTION_GET_CONTENT)
             intent.type = "*/*"
             intent.putExtra(Intent.EXTRA_MIME_TYPES, filter)
+            intent.addCategory(Intent.CATEGORY_OPENABLE)
             startActivityForResult(intent, REQUEST_CODE_FOR_PERFORM_SAF)
         }
     }

